app/cmd/plugin_upgrade.go

```diff
@@ -25,7 +27,10 @@ var pluginUpgradeOption PluginUpgradeOption
 
 func init() {
 	pluginCmd.AddCommand(pluginUpgradeCmd)
-	pluginUpgradeCmd.Flags().StringArrayVarP(&pluginUpgradeOption.Filter, "filter", "", []string{}, "Filter for the list, like: name=foo")
+	pluginUpgradeCmd.Flags().StringArrayVarP(&pluginUpgradeOption.Filter, "filter", "", []string{}, i18n.T("Filter for the list, like: name=foo"))
+	pluginUpgradeCmd.Flags().BoolVarP(&pluginUpgradeOption.All, "all", "", false, i18n.T("upgrade all plugins"))
```

Comment:
I believe I must missed this comment.
`the description should be a complete sentence which meanings you need to let your sentence starts with a capital.`